### DISCUSSION TITLE
NetRocks && SSH (connect to Armbian)

### DISCUSSION THREAD

**Discussion by verasminsk** [elfmz/far2l#1047] [Q&A]
не получается подключиться к машине через NetRocks, в консоле все ок:
![image](https://user-images.githubusercontent.com/18266668/128199831-625af00c-0c82-4cfd-8c02-463be303071f.png)

в от через фар не получается..

![image](https://user-images.githubusercontent.com/18266668/128200158-f4ad0750-e44b-4395-8911-0f09480a1b8c.png)
![image](https://user-images.githubusercontent.com/18266668/128200209-06914272-ff49-4776-9069-05149ba5640b.png)

![image](https://user-images.githubusercontent.com/18266668/128200268-22d3458d-a358-4f9b-aa9d-de4841db672f.png)

что не так может быть?

**Comment by elfmz** [comment]
Видел такие проблемы изза старой libssh vs очень новый ssh сервер, в результате чего старая libssh не умеет в криптоалгоритмы которые требуются для подключения к серверу.
Если дело в этом, то фиксить только обновлением libssh.

**Comment by elfmz** [comment]
Еще буквально сам недавно увидел - если ssh сервер делает reverse dns lookup что замедляет время подключения свыше таймаута (10 секунд по дефолту) - то получается такая вот ошибка. Стоит попробовать в Connection settings/Protocol options/Connection timeout увеличить до 30 сек.

**Comment by elfmz** [comment]
Если еще актуально, то после https://github.com/elfmz/far2l/commit/c99fbdb382250828d974edd8ce12ffa84cb4f817 можно попробовать сменить тип авторизацию с юзер-пассворд на интерактивный логин. Еще не очень понятно чем они различаются, есть подозрение что интерактивный менее секурен так как отправляет серверу пароль. Тем не менее как выяснилось бывают серверы которые поддерживают только интерактивный логин , а юзерпассворд - не поддерживают.

**Comment by m32** [comment]
In the first picture, you logged in via ssh, and in the far2l settings you want to use scp?
Change the protocol to sftp (second line in far2l connection settings), on debian it works without problems
